### PR TITLE
WIP: Start the process of gogo removal

### PR DESCRIPTION
**Pull request by sdake**
This updates Istio to run with protobuf 1.4.2 and brings in the API from 1.18 protobufs

The driver for this work is reflection - which is added to protobuf 1.4 series.

Related PR: https://github.com/istio/api/pull/1607

### CODE REVIEW

**Comment by sdake** [comment]
https://github.com/istio/istio/issues/26010

File: common-protos/google/api/expr/v1beta1/decl.proto [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Google LLC.
+// Copyright 2019 Google LLC.
```
Comment (ericvn) [inline common-protos/google/api/expr/v1beta1/decl.proto:1]: I see a mix of 2020 and 2019 (at least this one) copyrights. I believe I have seen the year being removed from copyrights for files in the istio repos.

Comment (howardjohn) [reply common-protos/google/api/expr/v1beta1/decl.proto:1]: these are vendored files so I think we just copy them as is.

Comment (ericvn) [reply common-protos/google/api/expr/v1beta1/decl.proto:1]: Makes sense.

**Comment by sdake** [comment]
@howardjohn and I spoke about him taking over this set of PRs. The work is fairly close to done. I will know more next week if this is something I will have bandwidth to continue working on.

Thank you for the reviews.

**Comment by jacob-delgado** [comment]
gogo has been removed for a while now. Closing.

**Pull request closed without merging**